app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.comments_counter(post)
+    where(post:).count
+  end
+
+  def self.update_comments_counter(post)
+    post.update(comments_counter: comments_counter(post))
+  end
```

Comment:
- [ ] Kindly change this to an instance method using the above suggestions.

- [ ] **[OPTIONAL]** I suggestion you add an `after_save` callback that triggers this method using the above suggestion.